### DISCUSSION TITLE
I have this error: ENOENT: no such file or directory, open '/root/.flowise/settings.json'

### DISCUSSION THREAD

**Discussion by remixdance** [FlowiseAI/Flowise#1581] [Q&A] [answered]
I updated my Open AI assistant and when try to chat The bot give me the error : ENOENT: no such file or directory, open '/root/.flowise/settings.json' How I can solve it 

here is my flow I have 3 custom tools , each tool  calls an API 
![image](https://github.com/FlowiseAI/Flowise/assets/42038961/f0fc2780-300c-46cc-8023-e1f8921d86ff)

I hope can help me
Im using render and flowise 1.4.10

**Comment by dkindlund** [comment, marked as answer]
Fixed in latest rev:
https://github.com/FlowiseAI/Flowise/issues/1579
